Casks/bootstrap-studio.rb

```diff
@@ -0,0 +1,23 @@
+cask 'bootstrap-studio' do
+  version '5.0.3'
+  sha256 'd208d5bb2b87ec1ea65ab2017317f35050a8d13c96e5bf6106817ef47c1fea6f'
+
+  url "https://bootstrapstudio.io/releases/desktop/#{version}/Bootstrap%20Studio.dmg"
```

Comment:
```suggestion
  url "https://bootstrapstudio.io/releases/desktop/#{version}/Bootstrap%20Studio.dmg"
  appcast "https://bootstrapstudio.io/app/updates/latest-mac.yml"
```
The app is trying to fetch updates from given URL but it results in 404 😩

Reply:
That file has no content

Reply:
I'm aware of that but that is an error on the upstream side. As soon as you run Bootstrap Studio it tries to download it.

Reply:
From the documentation:

> An appcast can be any URL hosted by the app’s developer that changes every time a new release is out or that contains the version number of the current release (e.g. a download HTML page). Webpages that only change on new version releases are preferred, as are sites that do not contain previous version strings (i.e. avoid changelog pages if the download page contains the current version number but not older ones). (Example: razorsql.rb)

The download page, https://bootstrapstudio.io/download, changes every time a release is published. Maybe that would be better? Or do you think I should just merge https://bootstrapstudio.io/app/updates/latest-mac.yml as you suggested earlier?

Reply:
Even https://bootstrapstudio.io/pages/releases changes

Reply:
Personally I would go with https://bootstrapstudio.io/pages/releases but that is question for @vitorgalvao